dm1_registers.xml

```diff
@@ -58,9 +58,9 @@
         </field>
     </register>
 
-    <register name="Bus Access Control" short="buscs" address="0x02">
+    <register name="Bus Access Control" short="sbcs" address="0x02">
```

Comment:
Does sbcs stand for System Bus Control/Status? If so, can you fix the long name (or at least add System to the front)?